packages/notification-services-controller/src/NotificationServicesPushController/NotificationServicesPushController.ts

```diff
@@ -213,6 +213,27 @@ export default class NotificationServicesPushController extends BaseController<
       return;
     }
 
+    this.#pushListenerUnsubscribe ??= await listenToPushNotifications({
+      env: this.#env,
+      listenToPushReceived: async (n) => {
+        this.messagingSystem.publish(
+          'NotificationServicesPushController:onNewNotifications',
+          n,
+        );
+        await this.#config.onPushNotificationReceived(n);
+      },
+      listenToPushClicked: (e, n) => {
+        if (n) {
+          this.messagingSystem.publish(
+            'NotificationServicesPushController:pushNotificationClicked',
+            n,
+          );
+        }
+
+        this.#config.onPushNotificationClicked(e, n);
+      },
+    });
```

Comment:
The context for this hotfix is to see if we can subscribe to push notifications independently from creating a new reg token (which requires auth, and will fail if wallet is locked)

If this fix works, I think we should do a little bit of polishing, since we want to subscribe when a user has a bearer token or doesn't have a bearer token.

Something like this:
```ts
// Handle creating new reg token (if available)
try {
  const bearerToken = await this.#getAndAssertBearerToken()
    .catch(() => null)
    
  // If there is a bearer token, lets try to refresh/create new reg token
  if (bearerToken) {
    // Activate Push Notifications
    const regToken = await activatePushNotifications({
      bearerToken,
      triggers: UUIDs,
      env: this.#env,
      createRegToken,
      platform: this.#config.platform,
    }).catch(() => null);

    if (regToken) {
      this.update((state) => {
        state.fcmToken = regToken;
      });
    }
  }
} catch () {
}

// Then subscribe to push notifications.
// If above was successful, we want to make sure that we subscribe to the new reg token
// If above was not successful (e.g. wallet was locked), we still want to subscribe to the existing reg token.
try {
  this.#pushListenerUnsubscribe?.();
  this.#pushListenerUnsubscribe ??= await listenToPushNotifications({ ... })
} catch () {
}
```